src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -709,17 +709,21 @@ describe('lib/viewers/image/ImageViewer', () => {
 
     describe('handleAnnotationColorChange', () => {
         beforeEach(() => {
+            image.annotator = {
+                emit: jest.fn(),
+            };
             image.annotationModule = {
                 setColor: jest.fn(),
             };
             image.renderUI = jest.fn();
         });
 
-        test('should call setColor and renderUI', () => {
+        test('should call setColor and renderUI, and emit color', () => {
             const color = '#fff';
             image.handleAnnotationColorChange(color);
 
             expect(image.annotationModule.setColor).toBeCalledWith(color);
+            expect(image.annotator.emit).toBeCalledWith('annotations_stroke_set', { color: '#fff' });
```

Comment:
Can we reuse `ANNOTATOR_EVENT.setStroke`?

Reply:
Maybe we can just pass in `color` instead of redefining it on L726? It's defined on L722.